clang/test/SemaSYCL/remove-restriction-builtin-assume-aligned.cpp

```diff
@@ -0,0 +1,17 @@
+// RUN: %clang_cc1 -fsycl-is-device -internal-isystem %S/Inputs -sycl-std=2020 -verify -pedantic -fsyntax-only %s
+#include "sycl.hpp"
+
+using namespace sycl;
+queue q;
+
+int main() {
+  int *Ptr[2];
+  // expected-no-diagnostics
+  q.submit([&](handler &h) {
+    h.single_task<class kernelA>([=]() {
+    int *APtr = (int *)__builtin_assume_aligned(Ptr, 32);
```

Comment:
Indentation?  clang-format didn't complain?  :-)

Reply:
Lint is green. Is it broken?

Reply:
It was a couple of days ago, but has been fixed since, I believe.

Reply:
Looks like it's using the configuration from this PR rather than the origin/sycl. Since this branch isn't up-to-date with the trunk, the buggy task was executed resulting in Lint not working.

@apstasen, any idea if/how that should be addressed in future?

Reply:
As for this PR, I'd suggest to merge origin/sycl into it to restart the testing with proper Lint.